### PR TITLE
Fixing nb_helloworld

### PR DESCRIPTION
**Pull request by malev**
@ijstokes @mmarchetti the instructions in the README are working fine now!

### CODE REVIEW

File: conda.recipe/post-link.bat [hunk 1]
```diff
@@ -1 +1 @@
-"%PREFIX%\bin\python" -m nb_helloworld.setup install --enable --prefix="%PREFIX%"
+"%PREFIX%/bin/python" -m nb_helloworld.setup install --enable --prefix="%PREFIX%"
```
Comment (mmarchetti) [inline conda.recipe/post-link.bat]: this will make Windows unhappy

File: conda.recipe/pre-unlink.bat [hunk 2]
```diff
@@ -1 +1 @@
-"%PREFIX%\bin\python" -m nb_helloworld.setup remove --prefix="%PREFIX%"
+"%PREFIX%/bin/python" -m nb_helloworld.setup remove --prefix="%PREFIX%"
```
Comment (mmarchetti) [inline conda.recipe/pre-unlink.bat]: this too

File: README.md [hunk 3]
```diff
@@ -1,48 +1,36 @@
-# Hello World Jupyter Extension
```
Comment (mmarchetti) [inline README.md]: Do we really want to remove all this information?

Comment (malev) [inline README.md]: we don't :smile: fixed

File: .binstar.yml [hunk 4]
```diff
@@ -3,14 +3,18 @@ package: nb_helloworld
 platform:
   - linux-64
   - osx-64
- # - win-64
+  - win-64
```
Comment (mmarchetti) [inline .binstar.yml]: We should consider win-32 and linux-32 for all of our extensions

**Comment by mmarchetti** [comment]
This looks good. Thanks Marcos!

**Pull request merged**